htdocs/core/class/commonobject.class.php

```diff
@@ -5671,7 +5671,11 @@ protected function commonGenerateDocument($modelspath, $modele, $outputlangs, $h
 					$update_main_doc_field = 1;
 				}
 
-				$this->indexFile($destfull, $update_main_doc_field);
+				// Check that the file exists, before indexing it.
+				// Hint: It does not exist, if we create a PDF and auto delete the ODT File
+				if (file_exists($destfull)) {
```

Comment:
Does this is ok if we use dol_is_file instead of file_exists ?

Reply:
I changed it